### PR TITLE
PMM-12045 fixed grafana cli per changes

### PR DESCRIPTION
**Pull request by vyarosh**

### CODE REVIEW

File: pmm-tests/pmm-framework.sh [hunk 1]
```diff
@@ -2500,7 +2502,13 @@ setup_custom_ami_instance() {
   echo "Setting up AMI instance with Custom Configuration"
   sudo cp /srv/pmm-qa/pmm-tests/prometheus.base.yml /srv/prometheus/prometheus.base.yml
   sudo supervisorctl restart pmm-managed
-  sudo grafana-cli plugins install alexanderzobnin-zabbix-app
+  export PMM_SERVER_DOCKER_CONTAINER=$(docker ps --format "table {{.ID}}\t{{.Image}}\t{{.Names}}" | grep 'pmm-server' | awk '{print $3}')
+  export pmm_minor_v=$(get_minor_version ${PMM_SERVER_DOCKER_CONTAINER})
+  if [[ $pmm_minor_v -ge 39 ]]; then
```
Comment (saikumar-vs) [inline pmm-tests/pmm-framework.sh:2507]: Is the if block required?

File: pmm-tests/check_upgrade.sh [hunk 2]
```diff
@@ -51,9 +51,14 @@ if [ "$DISTRIBUTION" == "ami" ]; then
 		rpm -qa | grep pmm-dump-$PMM_VERSION
 		sudo supervisorctl status | grep victoriametrics | grep RUNNING
 		sudo supervisorctl status | grep vmalert | grep RUNNING
-		grafana-cli plugins ls | grep "vertamedia-clickhouse-datasource @ 2.4.4"
-		grafana-cli plugins ls | grep alexanderzobnin-zabbix-app
 		sudo victoriametrics --version | grep pmm-6401-v1.77.1
+	  if [ $(echo "$PMM_VERSION" | awk -F. '{print $2}') -lt 39 ]; then
+  		grafana-cli plugins ls | grep "vertamedia-clickhouse-datasource @ 2.4.4"
+	  	grafana-cli plugins ls | grep alexanderzobnin-zabbix-app
+    else
```
Comment (saikumar-vs) [inline pmm-tests/check_upgrade.sh:58]: sorry about the nitpick :), seems like the else block requires indenting

**Review by saikumar-vs — CHANGES_REQUESTED** [review]

**Review by saikumar-vs — COMMENTED** [review]
thanks for clarifying the block required, dropped comment.

File: pmm-tests/check_upgrade.sh (hunk 2 above)
Comment (vyarosh) [reply pmm-tests/check_upgrade.sh:58]: Yeah! Good catch. There are mixed indents in the file. I've converted all indents to whitespaces.

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/job/pmm2-upgrade-tests/30150/

**Review by saikumar-vs — APPROVED** [review]

**Pull request merged**